db/primary_migrate/20230831124437_add_sign_in_count_and_second_mfa_encouragement_dismissed_at_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddSignInCountAndSecondMfaEncouragementDismissedAtToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :sign_in_count, :integer, default: 0, null: false
```

Comment:
I'm going to put this back in draft since I think we can actually get this without adding a new column, specifically by joining to the `events` table and using the `sign_in_after_2fa` event, similar to what's done in `User#second_last_signed_in_at`:

https://github.com/18F/identity-idp/blob/885af33d951d2e1d237123a27c4a481bb5993302/app/models/user.rb#L393-L396

Reply:
Updated to use existing `events` and drop the new column  in 2737847.